types.go

```diff
@@ -7,6 +7,7 @@ import (
 // Params are the parameters that the HipChat plugin can parse.
 type Params struct {
 	Notify   bool            `json:"notify"`
+	Url      string          `json:"url"`
```

Comment:
`Url` should be `URL` https://github.com/golang/go/wiki/CodeReviewComments#initialisms